packages/utils/blueprint-cli/src/bundle/package-bundle.ts

```diff
@@ -0,0 +1,31 @@
+import * as cp from 'child_process';
+import path from 'path';
+import * as pino from 'pino';
+
+/**
+ * EXPERIMENTAL. This relies on 'zip' being available locally.
+ */
+export function packageBundle(
+  logger: pino.BaseLogger,
+  folderPathAbs: string,
+  outputPathAbs: string,
+  options: {
+    encrypt?: boolean;
+  },
+) {
+  try {
+    let zipCommand = [`zip ${path.basename(outputPathAbs)}`, `-r ${path.basename(folderPathAbs)}`, '-x **/.git/**'].join(' ');
```

Comment:
`tar` might provide better interoperability.

Reply:
Tar unfortunately does not support an native encryption standard. Which practically makes it kinda annoying to use. 

The idea here is that your password can be used if you manually double click to unzip your zip file.